### PR TITLE
CVE-2019-3836

### PR DESCRIPTION
**Pull request by lpardoRH**

### CODE REVIEW

File: 2019/3xxx/CVE-2019-3836.json [hunk 1]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2019/3xxx/CVE-2019-3836.json]: Invalid JSON file

**Comment by iamleot** [comment]
Hello Laura,

Laura Pardo writes:
> [...]
> https://github.com/CVEProject/cvelist/pull/1774.diff
> [...]

The description is missing the affected version, I think it can be
modified from:

 `It was discovered in gnutls upstream [...]'

to:

 `It was discovered in gnutls versions from 3.6.3 before 3.6.7 [...]'

Similarly I think that version_data can be adjusted (it seems that
versions affected are >=3.6.3<3.6.7).

Please also add to `reference_data':

 <https://www.gnutls.org/security-new.html#GNUTLS-SA-2019-03-27>


Thank you!

**Comment by iamleot** [comment]
I'm not completely sure regarding versions affected though, sorry!

At least according:

 <https://www.gnutls.org/security-new.html#GNUTLS-SA-2019-03-27>

it seems that >=3.6.4<3.6.7 are affected, while according:

 <https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-3836>

it's >=3.6.3<3.6.7.

**Comment by lpardoRH** [comment]
According to the upstream commit https://gitlab.com/gnutls/gnutls/commit/96e07075e8f105b13e76b11e493d5aa2dd937226
This issue has been fixed in version 3.6.7
I'm not sure about which versions are affected so I've only added the fixed in one

Thanks

**Comment by iamleot** [comment]
I have requested further information on bugzilla.redhat.com and I will try to add any possible information from there here if noone beat me (but I think this shouldn't be non-blocking, e.g. >=3.6.3<3.6.7 can be used ATM and eventually adjusted later if needed).

Thanks again!

**Comment by iamleot** [comment]
> Can you please change the description from:
> 
> "It was discovered in gnutls before version 3.6.7 upstream that there is an uninitialized pointer access in gnutls versions 3.6.3 or later which can be triggered by certain post-handshake messages."
> 
> to (a bit easier to read):
> 
> "It was discovered in gnutls since 3.6.3 and before version 3.6.7 that there is an uninitialized pointer access which can be triggered by certain post-handshake messages."

This description suggestion wasn't commented/addressed.

Any rationale about that?

(it's not very important but IMHO would be nice to apply to clarify a bit it.)


Thanks!

**Comment by iamleot** [comment]
> I have requested further information on bugzilla.redhat.com and I will try to add any possible information from there here if noone beat me (but I think this shouldn't be non-blocking, e.g. >=3.6.3<3.6.7 can be used ATM and eventually adjusted later if needed).

JFTR, Pedro updated the information on bugzilla.redhat.com and affected versions are >=3.6.4<3.6.7.

**Pull request merged**